### ISSUE TITLE
Compile Time Error: `min(int64, int64)` produces an answer which is not in the inputs

### ISSUE THREAD

**Issue by tinkerer-red** [YoYoGames/GameMaker-Bugs#8022]
### Description

the output of the compile time literal should be `0x8877665544332211` / `-8613303245920329199` but it is producing a number which isn't even in the inputs. Imagine this is just a rounding error.

### Steps To Reproduce

```gml
var _a1 = 0x1122334455667788; // 1234605616436508552
var _a2 = 0x8877665544332211; // -8613303245920329199
var _a3 = 0x7FFFFFFFFFFFFFFF; // 9223372036854775807
var _a4 = 0x5566778811223344; // 6153737367135073092
show_debug_message(min(_a1, _a2, _a3, _a4)) // -8613303245920329199
show_debug_message(min(0x1122334455667788, 0x8877665544332211, 0x7FFFFFFFFFFFFFFF, 0x5566778811223344)) // -8613303245920328704
```

### How reliably can you recreate this issue using your steps above?

Always

### Which version of GameMaker are you reporting this issue for?

2024.8.1 (Monthly), 2024.1100 (Betas)

### Are you running GameMaker from inside your Steam library?

None

### Sample Package Attached?

- [X] I have attached my bug package

### Sample Project Added?

- [X] I have included a small sample project

**Comment by rwkay** [comment]
Fixed in 2024.11

**Comment by scott-dunbar** [comment]
Verified fixedin  IDE v2024.1100.0.653  Runtime v2024.1100.0.679
